core/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -637,6 +637,10 @@ public Builder metaData(MetaData metaData) {
             return this;
         }
 
+        public MetaData metaData() {
```

Comment:
Its actually an old remnant from something else I was trying, its not used anywhere so I'm deleting it.